documentation/staging/content/userguide/managing-domains/domain-in-image/_index.md

```diff
@@ -2,15 +2,14 @@
 title = "Domain in Image"
 date = 2019-02-23T16:45:16-05:00
 weight = 4
+draft = true
 pre = "<b> </b>"
 +++
 
 {{% children style="h4" description="true" %}}
 
-{{% notice warning %}}
-Oracle strongly recommends storing a domain image as private in the registry.
-A container image that contains a WebLogic domain home has sensitive information
-including keys and credentials that are used to access external resources
-(for example, the data source password). For more information, see
-[WebLogic domain in container image protection]({{<relref "/security/domain-security/image-protection#weblogic-domain-in-container-image-protection">}}).
+
+{{% notice note %}}
+Oracle strongly recommends that images containing WebLogic domains
+be kept in a private repository.
```

Comment:
I think it's useful to include the reasoning behind the recommendation, and fine to duplicate wording of a similar warning that this pull has elsewhere.

Reply:
The doc you are looking at is draft = true; the one we removed. I did duplicate this note in the current doc, with the larger explanation.